tests/unit/framework/prepare_sources_for_testing.py

```diff
@@ -158,7 +160,11 @@ def prepare_autowraps(self, test_file_path, preprocessed_file_path):
         with open(preprocessed_file_path) as f:
             code = f.readlines()
             for function in functions_to_wrap:
-                if function.startswith("env_") or function.startswith("bug"):
+                if function.startswith("env_") or function.startswith("bug") \
+                   or function.startswith("memcpy"):
+                    # added memcpy function to list of ignored functions,
+                    # because earlier it was regular function
```

Comment:
I think you don't need to describe the previous implementation here - it is enough to state that this is a macro

Reply:
fixed